forcedotcom/salesforcedx-vscode-automation-tests
# Fixes for Apex Replay Debugger on remote

Makes sure apexReplayDebugger.e2e.ts successfully passes on remote GHA

How to test:
Go to the [End to End Tests Workflow](https://github.com/forcedotcom/salesforcedx-vscode/actions/workflows/e2e.yml) in salesforcedx-vscode repo
Paste the PR's branch (cristi/ard-on-remote) in the second field of the Run workflow dropdown (Set the branch to use for automation tests)
Choose apexReplayDebugger.e2e.ts test from the dropdown to select the test to run.
AC:
Test suite should pass.

Successful runs:
- [298](https://github.com/forcedotcom/salesforcedx-vscode/actions/runs/5862747595)
- [299](https://github.com/forcedotcom/salesforcedx-vscode/actions/runs/5862748433)
- [300](https://github.com/forcedotcom/salesforcedx-vscode/actions/runs/5862749111)

@W-13743985@
